webkit_server.py

```diff
@@ -513,9 +513,12 @@ def issue_command(self, cmd, *args):
     self._writeline(cmd)
     self._writeline(str(len(args)))
     for arg in args:
-      arg = str(arg)
+      if type(arg) != str and type(arg) != unicode:
+        arg = str(arg)
+      if type(arg) == unicode:
+        arg = arg.encode("utf-8")
       self._writeline(str(len(arg)))
-      self._sock.sendall(arg.encode("utf-8"))
```

Comment:
Only unicode strings need to be encoded as utf-8 to be sent as bytes to webkit server